cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/aggregation/GroupByField.java

```diff
@@ -20,12 +20,10 @@
 
 public class GroupByField extends Field {
   public GroupByField(String fieldName) {
-    this.fieldName = fieldName;
-    this.alias = null;
+    super(fieldName);
   }
 
   public GroupByField(String fieldName, String alias) {
-    this.fieldName = fieldName;
-    this.alias = alias;
+    super(fieldName, alias);
```

Comment:
nitpick: Also add toString that generates the proper class name `GroupByField{...`